### PR TITLE
FEAT: Add language switch button

### PR DESCRIPTION
**Pull request by AoGao-Kedoka**
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import StudentServicesPage from "./pages/StudentServices";
 import CSSAPage from "./pages/CSSA";
 import AboutUsPage from "./pages/AboutUs";
 import MainNavigation from './components/layout/MainNavigation';
+import GermanWebsite from './pages/DE/GermanWebsite'
```
Comment (jazli1999) [inline src/App.js:13]: It seems we use double quotes for most cases. We may also want to fix that for imports above and below this line.  
```suggestion
import GermanWebsite from "./pages/DE/GermanWebsite";
```

File: src/components/layout/MainNavigation.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#nav-dropdown{
+    color: black;
+    font-weight: bold;
+    background-color: white;
+	border:0;
```
Comment (jazli1999) [inline src/components/layout/MainNavigation.css:5]:
```suggestion
    border:0;
```

File: src/pages/DE/GermanWebsite.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const GermanWebsite = () => {
+	return (
+		<div>
+		</div>
```
Comment (jazli1999) [inline src/pages/DE/GermanWebsite.js:4]: Can add some hint text to indicate it's still under construction. Blank page may give the wrong impression of "there's something broken".

**Review by jazli1999 — COMMENTED** [review]
Looks good, only some suggestions regarding code style, which can be safely ignored. I'll approve this PR and leave the decision whether to incorporate them up to you.

**Comment by AoGao-Kedoka** [comment]
> Looks good, only some suggestions regarding code style, which can be safely ignored. I'll approve this PR and leave the decision whether to incorporate them up to you.

You didn't approve XD

**Review by jazli1999 — APPROVED** [review]

**Pull request merged**